### PR TITLE
Translate Reconciliation

### PR DESCRIPTION
**Pull request by gattigaga**
Hi guys, I was completely translating Reconciliation page. Please review this and give me a feedback. :smile:

### CODE REVIEW

File: content/docs/reconciliation.md [hunk 1]
```diff
@@ -109,11 +108,11 @@ If you implement it naively, inserting an element at the beginning has worse per
 </ul>
 ```
 
-React will mutate every child instead of realizing it can keep the `<li>Duke</li>` and `<li>Villanova</li>` subtrees intact. This inefficiency can be a problem.
+React akan mengubah setiap *child* daripada menyadari itu dapat menjaga keutuhan sub-pohon `<li>Duke</li>` dan `<li>Villanova</li>`. Ketidakefisienan ini bisa menjadi masalah.
 
-### Keys {#keys}
+### Kunci {#keys}
```
Comment (resir014) [inline content/docs/reconciliation.md]:
```suggestion
### Key {#keys}
```

As per the [glossary](https://github.com/reactjs/id.reactjs.org/wiki/Glosarium#k), *key* tidak perlu diterjemahkan.

File: content/docs/reconciliation.md [hunk 2]
```diff
@@ -109,11 +108,11 @@ If you implement it naively, inserting an element at the beginning has worse per
...
 
-In order to solve this issue, React supports a `key` attribute. When children have keys, React uses the key to match children in the original tree with children in the subsequent tree. For example, adding a `key` to our inefficient example above can make the tree conversion efficient:
+Untuk mengatasi masalah ini, React mendukung atribut `key`. Saat *children* memiliki kunci, React menggunakan kunci untuk mencocokan *children* di dalam pohon yang asli dengan *children* di dalam pohon selanjutnya. Sebagai contoh, menambahkan `key` pada contoh tidak efisien kita diatas dapat membuat konversi pohon efisien:
```
Comment (resir014) [inline content/docs/reconciliation.md]:
```suggestion
Untuk mengatasi masalah ini, React mendukung atribut `key`. Saat *children* memiliki *key*, React menggunakan *key* untuk mencocokan *children* di dalam pohon yang asli dengan *children* di dalam pohon selanjutnya. Sebagai contoh, menambahkan `key` pada contoh tidak efisien kita diatas dapat membuat konversi pohon efisien:
```

Idem.

File: content/docs/reconciliation.md [hunk 3]
```diff
@@ -128,30 +127,30 @@ In order to solve this issue, React supports a `key` attribute. When children ha
 </ul>
 ```
 
-Now React knows that the element with key `'2014'` is the new one, and the elements with the keys `'2015'` and `'2016'` have just moved.
+Sekarang React tahu bahwa elemen dengan kunci `'2014'` adalah yang baru, dan elemen dengan kunci `'2015'` dan `'2016'` telah dipindahkan.
```
Comment (resir014) [inline content/docs/reconciliation.md]:
```suggestion
Sekarang React tahu bahwa elemen dengan *key* `'2014'` adalah yang baru, dan elemen dengan *key* `'2015'` dan `'2016'` telah dipindahkan.
```

**Comment by gattigaga** [comment]
Hi @resir014, I already change the words "kunci" to "key".

**Pull request merged**